config/story/genesis-node.json

```diff
@@ -37,28 +37,28 @@
       "accounts": [
         {
           "@type": "/cosmos.auth.v1beta1.BaseAccount",
-          "address": "story1frwzrqunlnh0263hm93msp86hykx9j5aljvnyx",
```

Comment:
We probably don't want to change the validator key here? Since it's used by local nodes

Reply:
These new addresses are derived from EVM addrs. Current one is derived from public key.